src/app/options/extra/extra.component.ts

```diff
@@ -72,25 +84,74 @@ export class OptionsExtraComponent {
       let _imp = JSON.parse(e.target.result);
 
       if (_imp && _imp.misc && _imp.misc.schema && _imp.misc.schema === "1.0") {
-        _imp.bookmark && Object.assign(this.settings.config.bookmark, _imp.bookmark);
+        _imp.bookmark &&
+          Object.assign(this.settings.config.bookmark, _imp.bookmark);
         _imp.date && Object.assign(this.settings.config.date, _imp.date);
         _imp.design && Object.assign(this.settings.config.design, _imp.design);
         _imp.i18n && Object.assign(this.settings.config.i18n, _imp.i18n);
-        _imp.message && Object.assign(this.settings.config.message, _imp.message);
+        _imp.message &&
+          Object.assign(this.settings.config.message, _imp.message);
         _imp.misc && Object.assign(this.settings.config.misc, _imp.misc);
         _imp.order && Object.assign(this.settings.config.order, _imp.order);
-        _imp.quickLink && Object.assign(this.settings.config.quickLink, _imp.quickLink);
+        _imp.quickLink &&
+          Object.assign(this.settings.config.quickLink, _imp.quickLink);
         _imp.search && Object.assign(this.settings.config.search, _imp.search);
         _imp.time && Object.assign(this.settings.config.time, _imp.time);
-        this.importStatus = 'success';
-        this.ga.field('button.import', 'true');
+        this.importStatus = "success";
+        this.ga.field("button.import", "true");
       } else {
-        this.importStatus = 'error';
-        this.ga.field('button.import', 'false');
+        this.importStatus = "error";
+        this.ga.field("button.import", "false");
       }
     };
 
     reader.readAsText(files[0]);
-    input.value = '';
+    input.value = "";
+  }
+
+  loadData(e) {
+    this.countryLists();
+  }
+
+  addCountry() {
+    let elementId = this.shared.createID("covid");
```

Comment:
The ID name should be in all caps, "COVID" instead of "covid".. Arbitrary I know, but that's how I've been doing it :)